content/en/blog/2023-11-modelix-platform-release-23.2/index.md

```diff
@@ -0,0 +1,100 @@
+---
+date: 2023-11-02
+title: "Modelix Platform Release 23.2"
+---
+
+We are happy to announce the autumn release of the modelix platform: **`23.2`**
+
+The versions bundled in this release are:
+
+```
+modelixCoreVersion = 3.12.1
+incrementalVersion = 0.2.0
+buildToolsVersion = 1.1.0
+modelixMPSPatchVersion = 157
+```
+
+If you want to know how to use this platform in your Gradle build, check out the corresponding [documentation](https://docs.modelix.org/modelix/23.2/platform/howto/usage-platform.html).
+
+- [Feature Highlights](#feature-highlights)
+  - [ModelQL](#modelql)
+  - [Model Client as NPM Package (experimental)](#model-client-as-npm-package-experimental)
+  - [Vue.js Bindings (experimental)](#vuejs-bindings-experimental)
+  - [MPS Model Adapters](#mps-model-adapters)
+  - [Bulk Model Synchronization Plugin (experimental)](#bulk-model-synchronization-plugin-experimental)
+- [Development Insights](#development-insights)
+  - [New Approach to MPS-Related Components](#new-approach-to-mps-related-components)
+  - [Contribution Guide](#contribution-guide)
+  - [Marking Experimental Features](#marking-experimental-features)
+  - [Extending Automation](#extending-automation)
+- [What's To Come](#whats-to-come)
+
+# Feature Highlights
+
+## ModelQL
+
+The [ModelQL query language](https://docs.modelix.org/modelix/23.2/core/explanation/modelql.html) provides a dynamic way of loading parts of the model on demand while reducing the number of requests to a minimum.
+
+## Model Client as NPM Package (experimental)
+
+The previously existing [model client](https://github.com/modelix/modelix.core/tree/modelix-23.2/model-client) is now also published as [`@modelix/model-client` NPM package](https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/model-client/).
+
+The exposed API is minimal and experimental. 
+In the future, we want to make more features from the Kotlin model client directly accessible from the exposed API in the NPM package.
+
+## Vue.js Bindings (experimental)
+
+Built upon [`@modelix/model-client`](https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/model-client/) we created a library to interact with models through [Vue.js](https://vuejs.org/).
+
+With [`@modelix/vue-model-api`](https://docs.modelix.org/modelix/23.2/core/reference/component-vue-model-api.html) we are currently building sample projects with Vue.js.
+After that, we want to stabilize the API and show you how you can use it for your projects.
+
+## MPS Model Adapters
+
+We created [adapters](https://github.com/modelix/modelix.core/tree/modelix-23.2/mps-model-adapters) that serve as an interface between MPS and the modelix.
+They forward changes made to a model through the [model-api of modelix](https://docs.modelix.org/modelix/23.2/core/reference/component-model-api.html) to the [Open API of MPS](https://www.jetbrains.com/help/mps/open-api-accessing-models-from-code.html) of MPS and vice versa.
```

Comment:
```suggestion
They forward changes made to a model through the [model-api of modelix](https://docs.modelix.org/modelix/23.2/core/reference/component-model-api.html) to the [Open API of MPS](https://www.jetbrains.com/help/mps/open-api-accessing-models-from-code.html) of MPS.
```
The other direction is not available (yet) and might be limited compared to the primary one.

Reply:
Done